data/json/vehicleparts/motor.json

```diff
@@ -23,15 +23,11 @@
     "looks_like": "motor_small",
     "difficulty": 1,
     "durability": 80,
-    "power": 750,
-    "energy_consumption": 854,
+    "power": 800,
```

Comment:
should be 780 W at 1,492 W consumption, the same as the car alternator running in reverse.

Reply:
That is 52% efficiency, which is extremely low for an electric motor!

Reply:
Yeah, further research leads me to believe that alternators are **far** less efficient at generating power than electric motors are at turning that power into motion.